README.md

```diff
@@ -1 +1,186 @@
-# Portfolio-setup-and-mobile-first
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+ <!-- LOGO -->
+
+  
+  <br/>
+
+<!-- MAIN HEADING -->
+
+  <h3><b>Portfolio</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
```

Comment:
[OPTIONAL] 
- [ ] For the `Live Demo`, make sure once you delete a section or a part from the `README`, you should delete it also from the table of contents, likewise when you re-add it back.